src/hw/raspi4/vectors.s

```diff
@@ -0,0 +1 @@
+/* Empty file to keep the build system happy. */
```

Comment:
The way our cmake setup is done it's very "top level file knows about everything".

The proper way is to have subdirs have their own cmakelists and they can decide what source files get added. But that's not relevant right now, just a thing I've been thinking about.

Reply:
I've fixed this now, you can choose which files to add so you don't need this dummy file. See src/hw/aarch64_virt/CMakeLists.txt for how to do it.